components/Button/button.css

```diff
@@ -0,0 +1,57 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+.btn{
+    /* min-width: 6rem; */
+    width: max-content;
+    font-size: 1rem;
+    font-weight: 500;
+    padding: 0.8rem 1rem 0.8rem 1rem;
+    margin: 1rem;
+    border: none;
+    border-radius: 3px;
+    background-color: var(--primary);
+    color: var(--font-secondary);
+}
+.btn:hover{
+    cursor: pointer;
+    background-color:var(--accent-hover) ;
+}
+.btn-primary{
+    background-color: var(--alert-primary);
+}
+.btn-primary:hover{
+    background-color: var(--alert-primary-hover);
+}
+.btn-secondary{
+ background-color: var(--secondary);
+ color:var(--primary);
+}
```

Comment:
You can use a code formatter like Prettier to format your code so that the code will have consistent spacing and formatting.